crates/client/rpc-core/src/lib.rs

```diff
@@ -27,9 +27,26 @@ use starknet_core::types::{
 #[derive(Serialize, Deserialize)]
 pub struct Felt(#[serde_as(as = "UfeHex")] pub FieldElement);
 
+#[derive(Serialize, Deserialize)]
+pub struct PredeployedAccount {
+    pub contract_address: FieldElement,
+    pub contract_class: ContractClass,
+    pub balance: FieldElement,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct PredeployedAccountsInfo {
+    pub accounts: Vec<PredeployedAccount>,
+    pub private_key: FieldElement,
+}
+
```

Comment:
Why is there a single private_key when multiple accounts can be deployed?
I would expect the possibility to fill in one for each account

Reply:
I can change that np